lib/resources/r4/account.yaml

```diff
@@ -0,0 +1,69 @@
+---
+name: Account
+field_name_base_url: http://hl7.org/fhir/r4/account-definitions.html#Account
+fields:
+- name: identifier
+  required: 'No'
+  cardinality: 0..*
+  type: Identifier
+  description: A business identifier of the account.
+  action: terminology
+  url: https://hl7.org/fhir/R4/account-definitions.html#Account.identifier
+  binding:
+    description: A business identifier of the account.
+    terminology:
+    - display: PFT Bill Alias Type
+      system: https://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/28200
+      info_link: https://fhir.cerner.com/millennium/r4/proprietary-codes/#code-set-28200-pft-bill-alias-type
+    - display: Account Number
+      system: https://fhir.cerner.com/accountnumber
+      info_link: .
+
+- name: type
+  required: 'No'
+  cardinality: 0..1
+  type: CodeableConcept
+  description: Categorizes the account for searching purposes.
+  action: terminology
+  url: https://hl7.org/fhir/R4/account-definitions.html#Account.type
+  binding:
+    description: Categorizes the account for searching purposes.
+    terminology:
+    - display: Account Types
+      system: http://terminology.hl7.org/CodeSystem/v3-ActCode
+      info_link: https://hl7.org/fhir/R4/valueset-account-type.html
+    - display: Account Types
+      system: https://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/18736
+      info_link: https://fhir.cerner.com/millennium/r4/proprietary-codes/#code-set-18736-account-types
+
+- name: Account-balance
```

Comment:
the account-balance extension has valueMoney which has valueMoney.code which is what this terminology applies to, seems like this needs to be more specific

Reply:
https://github.com/cerner/fhir.cerner.com/pull/291/commits/9abce72ad48f0c4d326b053a4dac4c5eb37a915c#diff-5a59138a5bd3eaa9ab30d592af99e473R39